pkg/systemd/quadlet/quadlet.go

```diff
@@ -655,6 +663,52 @@ func ConvertVolume(volume *parser.UnitFile, name string) (*parser.UnitFile, erro
 		opts.WriteString(fmt.Sprintf("gid=%d", gid))
 	}
 
+	if volume.HasKey(VolumeGroup, KeyCopy) {
+		copy, ok := volume.LookupBoolean(VolumeGroup, KeyCopy)
+		if ok {
+			if copy {
+				podman.add("--opt", "copy")
+			} else {
+				podman.add("--opt", "nocopy")
+			}
+		}
+	}
+
+	devValid := false
+
+	if volume.HasKey(VolumeGroup, "Device") {
+		dev, ok := volume.Lookup(VolumeGroup, KeyDevice)
+		if ok && len(dev) != 0 {
+			podman.add("--opt", fmt.Sprintf("device=%s", dev))
+			devValid = true
+		}
+	}
+
+	if volume.HasKey(VolumeGroup, KeyType) {
```

Comment:
In the following two cases, while there is some reason to check for the existence of the key, again, `ok` gives you the same functionality without having to doing two lookups